packages/cli/src/commands/help.ts

```diff
@@ -273,12 +278,15 @@ export function buildCommandExampleLines(command: Command) {
     }
     outputArray.push('');
   }
-  // delete the last newline added after examples iteration
-  outputArray.splice(-1);
 
   return outputArrayToString(outputArray);
 }
 
+function buildDescriptionLine(command: Command) {
+  const line: string[] = [INDENT, command.description];
```

Comment:
Let's skip indenting the description. When the description is long, it wraps and looks bad.

Reply:
I like the spacing personally. Perhaps for a long description we should automatically add newlines at the terminal width so that we can add spacing on subsequent lines to keep things aligned.

Reply:
I'm currently playing with https://www.npmjs.com/package/word-wrap and https://www.npmjs.com/package/cli-table3 to see if I can get more consistent indenting with less custom code. But it might be a fool's errand.